debug/debug.go

```diff
@@ -130,12 +153,14 @@ func Log(tag string, f string, args ...interface{}) {
 		f += "\n"
 	}
 
+	formatString := fmt.Sprintf("[% 25s] %-20s %s", tag, getPosition(), f)
```

Comment:
instead of 25, you could take the maximum width of all the tags that are specified...